modules/indexer/code/internal/util.go

```diff
@@ -48,3 +48,11 @@ func FilenameMatchIndexPos(content string) (int, int) {
 	}
 	return 0, len(content)
 }
+
+func ParseKeywordAsPhrase(keyword string) (string, bool) {
+	if strings.HasPrefix(keyword, `"`) && strings.HasSuffix(keyword, `"`) {
```

Comment:
How do we handle the case `keyword == "`?
I think the best approach would be to add the extra check `&& len(keyword) > 1`

Reply:
Good catch d51d73602bfb14b9b14b3b53db116b58f4163909